### PR TITLE
Wc crew smash

### PR DESCRIPTION
**Pull request by wcampbell91**

### CODE REVIEW

File: src/javascripts/components/flightDashboard/singleFlight/singleFlight.js [hunk 1]
```diff
@@ -6,7 +6,8 @@ const viewSingleFlight = (e) => {
   const flightId = e.target.closest('.flight-card').id;
   smash.getSingleFlightInfo(flightId)
     .then((flight) => {
-      const domString = `
+      console.error(flight.crew);
```
Comment (luketlancaster) [inline src/javascripts/components/flightDashboard/singleFlight/singleFlight.js:9]: 👀👀👀

File: src/javascripts/components/flightDashboard/singleFlight/singleFlight.js [hunk 2]
```diff
@@ -16,13 +17,22 @@ const viewSingleFlight = (e) => {
   <ul class="list-group list-group-flush">
     <li class="list-group-item">Origin: ${flight.origin.data.name}</li>
     <li class="list-group-item">Destination: ${flight.destination.data.name}</li>
```
Comment (luketlancaster) [inline src/javascripts/components/flightDashboard/singleFlight/singleFlight.js:19]: If this were me I’d refactor this so you don’t need to do `thing.data.property` (So change the smash to make flight.plane = response.data or whatever)

**Review by luketlancaster — COMMENTED** [review]
Couple of Luke Thoughts, take em or leave em!

**Review by jeaninedavis — APPROVED** [review]
👍

**Pull request merged**